src/isolation/filemap.rs

```diff
@@ -126,6 +168,18 @@ impl UhyveFileMap {
 		})
 	}
 
+	/// Get flags that should be appended to [`crate::hypercall::open`]
+	/// as per the structure's defined I/O mode.
+	#[inline]
+	#[cfg(target_os = "linux")]
+	pub(crate) fn get_io_mode_flags(&self) -> i32 {
+		match self.iomode {
```

Comment:
It would be nice if this was a method on `UhyveIoMode` (and this method would just forward to that).